example/README.md

```diff
@@ -231,34 +233,40 @@ Workflow tests are located in the `/spec/<workflow-name>` folder. Any tests and
+The workflow should be configured as it would be for a normal Cumulus deployment in a workflows terraform file. It must be deployed to the current deployment if testing locally.
 
-The workflows yaml files are located in the `/workflows/` folder and are split up to make the workflows easier to find and understand. When adding a new file, make sure to update the `app/config.yml` `stepFunctions` field.
+The workflows terraform files are located in the `/example/cumulus-tf` folder and are split up to make the workflows easier to find and understand.
 
 A new folder should be added in the `/spec` folder for the workflow and the tests should go into that folder with the input JSON files.
 
-Ideally the test can run in parallel with other tests and should be put in the `parallel` folder. If it cannot be, it should go in the `spec` folder. Only if the test should be run outside of the test suite should it go in the `standalone` folder.
+Ideally the test can run in parallel with other tests and should be put in the `parallel` folder. If it cannot be, it should go in the `spec/serial` folder. Only if the test should be run outside of the test suite should it go in the `standalone` folder.
 
 ## Fake data server
 
 A fake server is required for tests testing FTP/HTTP/HTTPS discover and downloads. The fake server should be set up once per account.
 
-The Cloudformation template for the fake data server is in `fake-server.yml`. To setup the fake server run:
+The Cloudformation template for the fake data server is in `fake-provider-cf.yml`. To setup the fake server, you can use the AWS CLI or AWS Console.
+
+If you want to use the AWS CLI, run:
 
 ```bash
-aws cloudformation deploy --template-file fake-server.yml --stack-name <stack-name> --parameter-overrides VpcId=<vpc-XXXXX> SubnetId=<subnet-XXXXXX> AZone=<az-zone> Ngap=true --capabilities CAPABILITY_NAMED_IAM
+aws cloudformation deploy --template-file fake-provider-cf.yml --stack-name <stack-name> --parameter-overrides VpcId=<vpc-XXXXX> Subnet=<subnet-XXXXXX> PermissionsBoundary=<permissions-boundary> NGAPProtAppInstanceMinimalPolicyName=<policy-name> LatestAmiId=<ami-id> FtpPassword=<ftp-password> Bucket=<bucket-name> Prefix=<prefix> --capabilities CAPABILITY_NAMED_IAM
 ```
 
 with the following parameters:
 
 - stack-name - Stack name for the fake server
 - VpcId - VPC ID
-- SubnetId - Subnet ID
-- AZone - Availability zone, needs to match the Subnet ID's availability zone
-- Ngap - `true` if in an NASA NGAP environment, will add the NGAP permission boundary to the IAM role created
+- Subnet - Subnet ID
+- PermissionsBoundary - A permissions boundary from NGAP. 
+- NGAPProtAppInstanceMinimalPolicyName - Will be included in the list of Amazon Resource Names (ARNs) of the IAM managed policies we want to attach to the user.
+- LatestAmiId - The most recent Amazon Machine Image ID
+- FtpPassword -
+- Bucket - 
```

Comment:
Should just be "S3 bucket name", because I believe it can be any bucket.

Reply:
Is there a reason that "internal" is used frequently for the value of bucket?

Reply:
Do you mean why is the "internal" bucket frequently used as the parameter value? If so, I think it's arbitrary, because usually we use our "internal" bucket for internal configuration such as for the fake provider. Also, the "internal" bucket shouldn't ever be written to by workflows, so it is "safer"